rest/reference/market_test.go

```diff
@@ -20,33 +20,33 @@ func TestGetMarketHolidays(t *testing.T) {
 	{
 		"exchange": "NYSE",
 		"name": "Thanksgiving",
-		"date": "2020-11-26T00:00:00.000Z",
```

Comment:
hmm, wasn't this copied directly from our api response? i feel like we should conform to that instead of changing the test

Reply:
cool i'm fine with it as long as we unmarshal the real responses correctly

Reply:
yeah I checked prod and currently every `date` in the response has the YYYY-MM-DD format